<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RTColorWidget</class>
 <widget class="QWidget" name="RTColorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>177</width>
    <height>121</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QComboBox" name="colorComboBox"/>
   </item>
   <item>
    <widget class="QWidget" name="rgbContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Red</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="redSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Green</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="greenSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Blue</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="blueSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Hex:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hexInput">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>colorComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>RTColorWidget</receiver>
   <slot>colorStringChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RTColorWidget</receiver>
   <slot>redChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RTColorWidget</receiver>
   <slot>greenChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RTColorWidget</receiver>
   <slot>blueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hexInput</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RTColorWidget</receiver>
   <slot>hexChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>colorChanged(Rayon::Color)</signal>
  <slot>colorStringChanged(QString)</slot>
  <slot>redChanged(int)</slot>
  <slot>greenChanged(int)</slot>
  <slot>blueChanged(int)</slot>
  <slot>hexChanged(QString)</slot>
 </slots>
</ui>
